src/components/MapContainer.js

```diff
@@ -19,6 +19,8 @@ import { drawBounds } from '../app/region-bounds'
 import { fetchDataTiles } from '../app/data'
 import { showRegion } from '../app/region'
 
+const PROGRAMMER_DEFINED_CONSTANT = 10
```

Comment:
OOPS MY BAD 😝  

I used `PROGRAMMER_DEFINED_CONSTANT` as a placeholder name, not something you should literally use! I think you should choose a name that's more descriptive of what it actually means!

Reply:
Oh haha my bad. Let me change that.